### DISCUSSION TITLE
Pressing Backspace in Media Data Pop-up rich text scrolls page

### DISCUSSION THREAD

**Discussion by AlessioGr** [payloadcms/payload#925] [General]
https://user-images.githubusercontent.com/70709113/183104277-c1801d2a-30a8-46e8-8cf4-0ce56c43bd86.mov

**Comment by jmikrut** [comment]
Hey @AlessioGr — this looks to be a Chromium bug not related to Payload:

https://www.reddit.com/r/chrome/comments/u03aqk/facebook_scrolling_cause_of_backspace/
https://www.reddit.com/r/edge/comments/syb76s/backspace_key_in_facebook_on_edge_makes_my_window/

Super weird.

Can you replicate by doing the following:

1. Go to https://demo.payloadcms.com/admin
2. Log in, go to Pages, go to Home
3. Switch tabs to Page Layout
4. In the first block, there's a Media upload element. Click Edit
5. In the rich text within the Edit modal, hit backspace

Does it cause scrolling? For me, it does not!

I'm gonna bump this to a discussion for now because I don't believe that there's much we can do about it. But, let's keep an eye on it because this is super strange.
